client/src/components/Upload.vue

```diff
@@ -39,20 +39,22 @@ export default {
         item => item.status !== "done" && item.status !== "error"
       ).length;
     },
-    // Takes the pending upload and returns the # of images or size of the file
+    /**
+     * Takes the pending upload and returns the # of images or size of the file based on the type and state
+     * @param {Object} pendingUpload - contains , status, type
+     *  size, and list of files to upload.
+     */
```

Comment:
```suggestion
    /**
     * @param {{ type: string, size: number, files: Array, uploading: boolean }} pendingUpload
     * @returns {number} # of images or size of file depending on type and state
     */
```

I noticed that your comment was out of date already.  `pendingUpload.status` was unused and `pendingUpload.uploading` wasn't mentioned.